### PR TITLE
[Feature] Debate type dashboard

### PR DESCRIPTION
**Pull request by Sqyphen**
## Description

Adds a list of all Debate types so that you can see all debates belonging to this type and find existing types to edit.

## Implementation

- Splits existing index page into a dashboard for debates and a placeholder homepage
- Adds new debate type listing page
- Adds new debates *by type* listing page

## Screenshot

n/a

## .env changes

n/a

## Additional requirements

n/a

Whaaaaa

### CODE REVIEW

File: models/dynamoDb.js [hunk 1]
```diff
@@ -312,13 +312,17 @@ async function createDebateType({
 	displayName,
 	createdBy
 }) {
+	const createdAt = date;
```
Comment (Lily2point0) [inline models/dynamoDb.js]: It doesn't look like there's a separate "edit debate type" function; in which case the created date will be overridden with each edit. Also a bit confused as to where `date` comes from?

File: routes/admin/debateType.js [hunk 2]
```diff
@@ -138,6 +138,38 @@ router.post('/edit/:debateName', async (req, res) => {
 	}
 });
 
+router.get('/list/:debateName', async (req, res) => {
```
Comment (Lily2point0) [inline routes/admin/debateType.js]: should it be `/list/:debateType`?

File: models/dynamoDb.js (hunk 1 above)
Comment (Sqyphen) [reply models/dynamoDb.js]: Oh dear...

Comment (Sqyphen) [reply models/dynamoDb.js]: This has been fixed now :)

File: routes/admin/debateType.js (hunk 2 above)
Comment (Sqyphen) [reply routes/admin/debateType.js]: Yup, fixed

**Pull request merged**